### DISCUSSION TITLE
Active Status dropdown

### DISCUSSION THREAD

**Discussion by naveeddil** [xlg8/APMIS-Project#666] [General]
Active Status and All Status are one and the same thing.

All = Active + Archived

![APMIS active status](https://github.com/AFG-Polio-Data/APMIS-Project/assets/54671296/af0978dd-68db-469b-aae7-98f9671034d8)

@RaziaMahmodi @bnygren

**Comment by RaziaMahmodi** [comment]
@naveeddil Dev team sis working on it.

**Comment by naveeddil** [comment]
Fixed
